android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt

```diff
@@ -23,30 +22,28 @@ class ScreenStack(context: Context?) : ScreenContainer<ScreenStackFragment>(cont
     private var reverseLastTwoChildren = false
     private var previousChildrenCount = 0
     var goingForward = false
+
     fun dismiss(screenFragment: ScreenStackFragment) {
         mDismissed.add(screenFragment)
         performUpdatesNow()
     }
 
     override val topScreen: Screen?
         get() = mTopScreen?.screen
+
     val rootScreen: Screen
         get() {
-            var i = 0
             val size = screenCount
-            while (i < size) {
+            for (i in 0 until size) {
```

Comment:
Similarly here, couldn't you use `screenCount` directly?

Reply:
[7aad6e6](https://github.com/software-mansion/react-native-screens/pull/1558/commits/7aad6e6a32178303c9905760644a2cd748911077)